doc/extensions.md

```diff
@@ -0,0 +1,73 @@
+# SkyPortal Extensions
+
+It is common for specific instances of SkyPortal to employ ``extensions,'' which use added components and APIs not available in the baseline app, usually because of particular forms of data access and distribution. Some examples include [Fritz](https://github.com/fritz-marshal/fritz) or [SkyPortal-GRANDMA](https://github.com/grandma-collaboration/icare).
+
+For the APIs, they will have a folder: extensions/skyportal/skyportal/handlers/api
+For the ducks, they will have a folder: extensions/skyportal/static/js/ducks
+For the components, they will have a folder: extensions/skyportal/static/js/components
+
+When deploying, each of these folders are copied into skyportal in their corresponding folder.
+
+For the APIs in particular, a dedicated app_server.py needs to be created. To deploy, it will take a form similar to [this](https://github.com/fritz-marshal/fritz/blob/main/extensions/skyportal/skyportal/app_server_fritz.py), where in addition to the baseline application loaded with
```

Comment:
```suggestion
For APIs in particular, a dedicated `app_server.py` needs to be created. To deploy, it will take a form similar to [this](https://github.com/fritz-marshal/fritz/blob/main/extensions/skyportal/skyportal/app_server_fritz.py), where in addition to the baseline application loaded with
```

And then you also need something like this to show how to hook up the factory:

> This is known as the "app factory", and is configured in the `config.yaml` as follows.